meson.build

```diff
@@ -0,0 +1,2005 @@
+conf.set('SIZEOF_DEV_T', cc.sizeof('dev_t', prefix : '#include <sys/types.h>'))
+conf.set('SIZEOF_INO_T', cc.sizeof('ino_t', prefix : '#include <sys/types.h>'))
+conf.set('SIZEOF_TIME_T', cc.sizeof('time_t', prefix : '#include <sys/time.h>'))
+conf.set('SIZEOF_RLIM_T', cc.sizeof('rlim_t', prefix : '#include <sys/resource.h>'))
+
+decl_headers = '''
+#include <uchar.h>
+#include <linux/ethtool.h>
+'''
+# FIXME: key_serial_t is only defined in keyutils.h, this is bound to fail
+
+foreach decl : ['char16_t',
+                'char32_t',
+                'key_serial_t',
+                'struct ethtool_link_settings',
+               ]
+  if cc.sizeof(decl, prefix : decl_headers) > 0
+    # We get -1 if the size cannot be determined
+    conf.set('HAVE_' + decl.underscorify().to_upper(), 1)
+  endif
+endforeach
+
+foreach decl : [['IFLA_INET6_ADDR_GEN_MODE',         'linux/if_link.h'],
+                ['IN6_ADDR_GEN_MODE_STABLE_PRIVACY', 'linux/if_link.h'],
+                ['IFLA_VRF_TABLE',                   'linux/if_link.h'],
+                ['IFLA_MACVLAN_FLAGS',               'linux/if_link.h'],
+                ['IFLA_IPVLAN_MODE',                 'linux/if_link.h'],
+                ['IFLA_PHYS_PORT_ID',                'linux/if_link.h'],
+                ['IFLA_BOND_AD_INFO',                'linux/if_link.h'],
+                ['IFLA_VLAN_PROTOCOL',               'linux/if_link.h'],
+                ['IFLA_VXLAN_REMCSUM_NOPARTIAL',     'linux/if_link.h'],
+                ['IFLA_VXLAN_GPE',                   'linux/if_link.h'],
+                # if_tunnel.h is buggy and cannot be included on its own
+                ['IFLA_VTI_REMOTE',                  'linux/if_tunnel.h', '#include <net/if.h>'],
+                ['IFLA_IPTUN_ENCAP_DPORT',           'linux/if_tunnel.h', '#include <net/if.h>'],
+                ['IFLA_GRE_ENCAP_DPORT',             'linux/if_tunnel.h', '#include <net/if.h>'],
+                ['IFLA_BRIDGE_VLAN_INFO',            'linux/if_bridge.h'],
+                ['IFLA_BRPORT_PROXYARP',             'linux/if_link.h'],
+                ['IFLA_BRPORT_LEARNING_SYNC',        'linux/if_link.h'],
+                ['IFLA_BR_VLAN_DEFAULT_PVID',        'linux/if_link.h'],
+                ['NDA_IFINDEX',                      'linux/neighbour.h'],
+                ['IFA_FLAGS',                        'linux/if_addr.h'],
+                ['LO_FLAGS_PARTSCAN',                'linux/loop.h'],
+               ]
+  prefix = decl.length() > 2 ? decl[2] : ''
+  have = cc.has_header_symbol(decl[1], decl[0], prefix : prefix)
+  conf.set10('HAVE_DECL_' + decl[0], have)
+endforeach
+
+skip = false
+foreach ident : ['secure_getenv', '__secure_getenv']
+  if not skip and cc.has_function(ident)
+    conf.set('HAVE_' + ident.to_upper(), 1)
+    skip = true
+  endif
+endforeach
+
+foreach ident : [
+  ['memfd_create',      '''#include <sys/memfd.h>'''],
+  ['gettid',            '''#include <sys/types.h>'''],
```

Comment:
I do not particularly care, how we indent things, just that it's consistent. Compare this with line 320
Or look at 1909, 1978,1985 compared to other places where we indent like
```
foo = executable(
                           bla
                          blub)